gprofiler/profilers/python.py

```diff
@@ -255,6 +262,12 @@ def stop(self):
             logger.info("Finished profiling Python processes with PyPerf")
 
 
+@register_profiler(
+    "Python",
+    possible_modes=["auto", "pyperf", "pyspy", "disabed", "none"],
```

Comment:
Comment on `none` that's it legacy.

Or actually - just add the legacy automatically in the shared code?